src/Recipe.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import style from "./recipe.module.css";
+
+const Recipe = ({ title, calories, image, ingredients }) => {
+  return (
+    <div className={style.recipe}>
+      <h1>{title}</h1>
+      <ol>
+        {ingredients.map((ingredient) => (
+          <li>{ingredient.text}</li>
+        ))}
+      </ol>
+      <p>{calories}</p>
```

Comment:
nice job on the mapping and clean code! one minor nitpick i have is for user experience, I would recommend rounding calories up to the nearest integer either by having some function that rounds the pure calorie info and outputs the nearest integer

Reply:
also on the note of user experience, it would be good to label what exactly this number represents (eg. `<p>Calories: {calories}</p>`) since it's displayed as just a number on the frontend which may be confusing.